certbot/certbot/_internal/client.py

```diff
@@ -644,7 +635,8 @@ def _recovery_routine_with_msg(self, success_msg):
 
         """
         self.installer.recovery_routine()
-        display_util.notify(success_msg)
+        if success_msg:
+            display_util.notify(success_msg)
```

Comment:
I think we should add an else here to say "Could not deploy certificate." just because it's a little bit of a strange transition into the next steps without it.